graylog2-server/src/main/java/org/graylog2/inputs/transports/KafkaTransport.java

```diff
@@ -187,7 +186,9 @@ public void run() {
         final Properties props = new Properties();
 
         props.put("group.id", configuration.getString(CK_GROUP_ID, DEFAULT_GROUP_ID));
-        props.put("client.id", "gl2-" + nodeId + "-" + input.getId());
+        props.put("client.id", "gl2-" + nodeId + "-" + input.getId(
+
+        ));
```

Comment:
@dennisoelkers looks like this slipped in

Reply:
Thanks! Fixed.